graders/python/python_autograder/pl_unit_test.py

```diff
@@ -0,0 +1,144 @@
+import unittest
+from types import FunctionType
+from collections import namedtuple
+from pl_helpers import (points, name, save_plot, print_student_code, not_repeated)
+from pl_execute import execute_code
+from code_feedback import Feedback
+
+# Needed to ensure matplotlib runs on Docker
+import matplotlib
+matplotlib.use('Agg')
+
+
+class PrairieLearnTestCase(unittest.TestCase):
```

Comment:
It's up to you and Nicolas, but I'd support a change to PL prefix even though it would mean changing all my code problems.  "Prairie" is just really hard to spell :slightly_smiling_face:

Reply:
Agreed, I like the "PL" prefix